app/src/sharedTest/java/org/oppia/app/testing/DragDropTestActivityTest.kt

```diff
@@ -0,0 +1,87 @@
+package org.oppia.app.testing
+
+import androidx.test.core.app.ActivityScenario.launch
+import androidx.test.espresso.Espresso
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.GeneralLocation
+import androidx.test.espresso.action.Press
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.matcher.ViewMatchers.withText
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.oppia.app.R
+import org.oppia.app.recyclerview.RecyclerViewMatcher.Companion.atPosition
+import org.oppia.app.utility.ChildViewCoordinatesProvider
+import org.oppia.app.utility.CustomGeneralLocation
+import org.oppia.app.utility.DragViewAction
+import org.oppia.app.utility.RecyclerViewCoordinatesProvider
+
+@RunWith(AndroidJUnit4::class)
+class DragDropTestActivityTest {
+
+  @Test
+  fun testDragDropTestActivity_dragItem0ToPosition1() {
+    launch(DragDropTestActivity::class.java).use {
+
+      safelyWaitUntilIdle()
```

Comment:
Is this actually needed? onIdle() should be called as part of the Espresso matchers below.

Reply:
removed